src/test/java/ru/training/at/hw5/RunnnerForCucumberTests.java

```diff
@@ -0,0 +1,9 @@
+package ru.training.at.hw5;
+
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(features = {"src/test/resources/features"})
```

Comment:
would be better to refer to it not as a source code, but as a classpath
```suggestion
@CucumberOptions(features = {"classpath:features"})
```